packages-next/keystone/src/lib/core/tests/List.test.ts

```diff
@@ -530,7 +531,9 @@ describe(`getGqlQueries()`, () => {
           orderBy: String
           first: Int
           skip: Int
-        ): _QueryMeta`,
+        ): _QueryMeta @deprecated(reason: \"This query will be removed in a future version. Please use allTestsCount instead.\")`,
+        `""" Count the number of items that match the where clause """
```

Comment:
Let's not have this description, it just clutters up the schema, the naming makes it clear what it does